tests/tensorflow/nightly/tf-resnet-imagenet.libsonnet

```diff
@@ -15,6 +15,7 @@
 local experimental = import '../experimental.libsonnet';
 local common = import 'common.libsonnet';
 local mixins = import 'templates/mixins.libsonnet';
+local timeouts = import 'templates/timeouts.libsonnet';
```

Comment:
Is this addition accidental?

Reply:
Oh thanks Will for catching it. It is accidental and used no where in the file.